test/test_torch.py

```diff
@@ -10382,6 +10337,108 @@ def run_test(tensor_dims, some):
         for tensor_dims, some in product(tensor_dims_list, [True, False]):
             run_test(tensor_dims, some)
 
+    @dtypes(torch.int, torch.long, torch.float, torch.double)
+    @dtypesIfCUDA(torch.int, torch.long, torch.half, torch.float, torch.double)
+    def test_median_real_values(self, device, dtype):
+        # Generate random 0-3D sizes
+        sizes = [torch.randint(1, 32, (i,)) for i in range(4) for _ in range(2)]
```

Comment:
You probably don't want to generate your sizes as a torch tensor that you're converting to a list. This should be easier to do using just Python.